platform/root/__tests__/index.test.ts

```diff
@@ -120,7 +120,7 @@ describe('when configuring logger fails', () => {
 
     mockServer.mockClear();
 
-    await root.start();
+    await expect(root.start()).rejects.toMatchSnapshot();
```

Comment:
__note:__ that's the second time when I see the issue in transpilation of `async ()`, but can't recall the issue number. So couldn't use `expect(async () => await root.start()).toThrowErrorMatchingSnapshot()` :/